src/herald/start_user_internet_available.py

```diff
@@ -0,0 +1,48 @@
+# start_internet_available_user.py
+#
+# Copyright (C) 2019 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Main program for start-user-internet-available script.
+
+
+import os
+
+from herald.utils import get_logged_in_users, start_user_target
+
+
+def start_internet_available_user_target(user):
+    """Start the internet-available-user.target for a given user."""
+
+    return start_user_target('internet-available-user.target', user)
+
+
+def start_internet_available_user_target_for_all_users():
+    """
+    Start the internet-available-user.target for all currently
+    logged in users.
+    """
+
+    users = get_logged_in_users()
```

Comment:
I don't understand the concept of "logged in users". There can only be 1 interactive user at a given time.
In other words... If I ssh into the kit, am I going to be `kano-synced` in the background?

Reply:
`man who` would provide more info here. The function returns a `set` of all users to remove duplicates. It's true that on our system there can be only 1 interactive user preset, in which case the function will return only that username.

Reply:
What if I `sudo bash` or if I login remotely as a different user? Or if I have a root shell for whatever reason? Sorry to disagree Radu, but I think we are opening a door to a bunch of possible problems. At least I'd suggest to filter out the `root` user for precaution.

I thought the intention was to `kano-sync` and launch other interactive-user related tasks - just not for anyone. But correct me if I am wrong.

Reply:
If one were to login remotely using a different account than the currently interactive one, your user would have the target raised which is exactly what users of this library would expect. 

Specifically to `kano-sync`, yes - it would be invoked for you as remote user if you were logged in to KW services, but only when the target would be raised. I think that is entirely expected and robust.

On the immediate line below to the one we're referring here, you can see I'm explicitly discarding `'root'` as a user from the `set` already.